package.json

```diff
@@ -147,5 +148,12 @@
         }
       }
     ]
+  },
+  "jest": {
+    "verbose": false,
+    "testEnvironment": "node",
+    "testMatch": [
+      "<rootDir>/test/**/*.test.js"
```

Comment:
Oh I see, this is only running the root test dir. I would prefer this be added to `test-unit` instead of overriding the global

Reply:
@styfle I rearranged this to remove the global jest config. What do you think?